application/Application.d.ts

```diff
@@ -0,0 +1,128 @@
+// There is no Interger in Typescript so it has been classed as a number but the Interger is still expected. 
+type Interger = number;
+
+declare class Application {
+  hostUrl: any;
+  apiKey: any;
```

Comment:
If `this.host.URL = host` does that make `hostURL` a string? and same goes for `apiKey`

Reply:
Yes, both are strings

Reply:
thanks